evalai/utils/challenges.py

```diff
@@ -0,0 +1,74 @@
+import os
+import json
+import requests
+
+from click import echo
+
+from pylsy import pylsytable
+
+from evalai.utils.auth import get_headers
+from evalai.utils.urls import urls
+from evalai.utils.common import valid_token
+
+
+API_HOST_URL = os.environ.get("EVALAI_API_URL", 'http://localhost:8000')
+
+
+def get_challenges(url):
+
+    headers = get_headers()
+    response = requests.get(url, headers=headers)
+    response_json = json.loads(response.text)
```

Comment:
The `requests` Response object has a method `.json()`. Any reason you're not using that?

Reply:
I was not aware of that. Thanks for pointing it out. I'll switch to that.